pandas/tests/scalar/period/test_asfreq.py

```diff
@@ -115,8 +115,6 @@ def test_conv_annual(self):
         assert ival_A.asfreq("H", "E") == ival_A_to_H_end
         assert ival_A.asfreq("min", "S") == ival_A_to_T_start
         assert ival_A.asfreq("min", "E") == ival_A_to_T_end
-        assert ival_A.asfreq("T", "S") == ival_A_to_T_start
-        assert ival_A.asfreq("T", "E") == ival_A_to_T_end
```

Comment:
could we just put these in a `tm.assert_produces_warning` context manager?

Reply:
I did it and got an `AssertionError: Did not see expected warning of class 'FutureWarning'`. Now I am trying to fix that.